content/posts/docker-prune.md

```diff
@@ -9,12 +9,23 @@ ShowToc: true
 draft: true
 ---
 
-Docker a is platform for developing, shipping and running applications in isolated, lightweight, portable containers. It is a critical part of a developer's toolbelt. I use it everyday.
+[//]: # (elementary question but is there direct coorelation between imaze size and consumed disk?)
```

Comment:
Yes, the larger the image, the more disk space it consumes.

We're not just talking about images though. There are many different Docker objects, such as images, containers, volumes, and networks.